genvm/core/gomock_reflect_1275411437/prog.go

```diff
@@ -0,0 +1,66 @@
+
+package main
```

Comment:
what's this file about?

Reply:
just noticed, seems like these are leftovers from `gomock`? I'll delete them.